src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -186,18 +187,26 @@ public WebElement forElementVisible(WebElement element) {
     }
   }
 
-  public WebElement forElementVisible(WebElement element, int timeout, int polling) {
+  public WebElement forElementVisible(WebElement element, int timeout_sec, int polling) {
     changeImplicitWait(250, TimeUnit.MILLISECONDS);
     try {
-      return new WebDriverWait(driver, timeout, polling).until(ExpectedConditions
+      return new WebDriverWait(driver, timeout_sec, polling).until(ExpectedConditions
                                                                    .visibilityOf(element));
     } finally {
       restoreDeaultImplicitWait();
     }
   }
 
-  public WebElement forElementVisible(WebElement element, int timeout) {
-    return forElementVisible(element, timeout, 500);
+  public WebElement forElementVisible(WebElement element, int timeout_sec) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)